### DISCUSSION TITLE
MobaXterm無法連線至Ubuntu虛擬機

### DISCUSSION THREAD

**Discussion by chloewangmeow** [bol-edu/HLS-SOC-Discussions#60] [Q&A]
教授、助教、各位同學好，我在使用ssh連線至虛擬機時，有輸入正確的IP位址，但一直連不上，並出現連線逾時的訊息（如圖）

![螢幕擷取畫面 2023-09-21 110336](https://github.com/bol-edu/HLS-SOC-Discussions/assets/76718011/41d50793-1997-49eb-ba07-6610819085fe)

我有檢查過虛擬機的網路介面卡確實有選擇VirtualBox Host-Only Ethernet Adapter

![螢幕擷取畫面 2023-09-21 110514](https://github.com/bol-edu/HLS-SOC-Discussions/assets/76718011/9dc08d04-c09e-47db-8996-775e574ad96b)

但在檢查主機端的DHCP時，發現並未顯示此網路介面卡

![螢幕擷取畫面 2023-09-21 110420](https://github.com/bol-edu/HLS-SOC-Discussions/assets/76718011/bddcb7ad-899f-49df-a15a-d1e522ab3af9)

因為一開始剛安裝好ubuntu時有成功ssh連線過，所以我自行猜測，有可能跟我曾經重新驅動過網路介面卡有關。

我在安裝vitis時，安裝到一半，網路介面卡因為不明原因突然從電腦消失，導致虛擬機無法開啟，因此我上網找尋解決方法重新驅動網路介面卡（到目錄 C:\Program Files\Oracle\VirtualBox\drivers\network\netadp6\VBoxNetAdp6.inf 右鍵安裝），有驅動網路介面卡，成功開啟虛擬機。

![螢幕擷取畫面 2023-09-21 115004](https://github.com/bol-edu/HLS-SOC-Discussions/assets/76718011/c70ccf83-6d7f-46f8-8d4d-a4710e19c5a4)

但在這之後，我便一直無法用ssh連線至虛擬機。
想請教有什麼可能的原因，以及成功連線的解決辦法。謝謝！

**Comment by kevinjantw** [comment]
目前VirtualBox主機端介面卡沒有出現在windows機器上(不確定裝那個VBoxNetAdp6.inf是正確?) 正常應該是像這個
<img width="468" alt="269246682-4086db14-94de-4f8a-8d61-ea0c7e1db307" src="https://github.com/bol-edu/HLS-SOC-Discussions/assets/11850122/3dfc8513-40c8-44c0-b51d-1239fe64b245">
請再確認這裡的設定
![269131212-c56d2fa3-3c52-41a3-bdfd-9b184aa25f11](https://github.com/bol-edu/HLS-SOC-Discussions/assets/11850122/26cf2fe2-04f8-40c7-adaf-448868625dc8)
enp0s8與主機端介面卡在相同網段後會相通
如果還沒有解決 建議VirtualBox及VM全部移除刪除後 重新安裝設定

**Reply by chloewangmeow** [reply]
了解，若還是沒解決我再重新安裝，謝謝您的回覆！
